docs/spec/spec.md

```diff
@@ -82,14 +82,15 @@ The conforming implementation of the specification is released and included in t
         * 8.1.1. [Request interceptor](#811-request-interceptor)
             * 8.1.1.1. [Request context](#8111-requestcontext)
             * 8.1.1.2. [Next method](#8112-next-method)
-            * 8.1.1.3. [Returning error](#8113-returning-error)
-        * 8.1.2. [Request error interceptor](#812-request-error-interceptor)
-        * 8.1.3. [Engaging interceptor](#813-engaging-interceptors)
-            * 8.1.3.1. [Service level](#8131-service-level)
-            * 8.1.3.2. [Listener level](#8132-listener-level)
-            * 8.1.3.3. [Execution order of interceptors](#8133-execution-order-of-interceptors)
-        * 8.1.4. [Data binding](#814-data-binding)
-        * 8.1.5. [Return to respond](#815-return-to-respond)
+            * 8.1.1.3. [Return to respond](#8113-return-to-respond)
+        * 8.1.2. [Response interceptor](#812-response-interceptor)
+            * 8.1.2.1. [Return to respond](#8113-return-to-respond)
+        * 8.1.3. [Request error interceptor and response error interceptor](#813-request-error-interceptor-and-response-error-interceptor)
+        * 8.1.4. [Engaging interceptor](#813-engaging-interceptors)
+            * 8.1.4.1. [Service level](#8131-service-level)
```

Comment:
Incorrect title links

Reply:
Check others as well

Reply:
Fixed in [87d6d3a](https://github.com/ballerina-platform/module-ballerina-http/pull/924/commits/87d6d3ad68a6b39bbdb54846fa7639979a30d8df)